### ISSUE TITLE
Benchmark test might be unreliable

### ISSUE THREAD

**Issue by marco-c** [mozilla/j2me.js#1294]
The benchmark samples might not be normally distributed, which makes the Student's t-test results unreliable.

This is a Q-Q plot of one run of the benchmark (30 rounds):
![qq plot](https://cloud.githubusercontent.com/assets/1616846/6766010/6e14dc04-cff5-11e4-8e1c-bf127f989ce5.png)
Obviously the data is not normally distributed in this case.

**Comment by brendandahl** [comment]
![screenshot 2015-03-23 10 34 39](https://cloud.githubusercontent.com/assets/942640/6786235/42621b18-d148-11e4-8ebb-dcd71f64fe84.png)

100round run^

Without the tails it's not far off from a normal distribution. There are a few statistician people in metrics near my desk. If I get some time a little later I'll discuss with them.

**Comment by marco-c** [comment]
Yeah, the more rounds the better it is for the central limit theorem (the magic number is often 30)
Maybe we should draw the quantile-quantile plot after running the benchmarks, so that we avoid wrong conclusions.

**Comment by marco-c** [comment]
(I don't have enough knowledge about statistics to tell if the tails are a problem)

**Comment by brendandahl** [comment]
![norm](https://cloud.githubusercontent.com/assets/942640/6791661/18d3875c-d16e-11e4-98ca-70e92d223bea.png)

Memory spikes seem to correlate to startup time spikes.
